datalad/support/annexrepo.py

```diff
@@ -1289,7 +1289,8 @@ def _init(self, version=None, description=None):
                                 where='local',
                                 reload=False)
 
-        self._run_annex_command('init', annex_options=opts)
+        self._run_annex_command('init', annex_options=opts,
+                                log_stdout=False, log_stderr=False, log_online=True)
```

Comment:
I do not think that it is a good idea to make a special case for `init` here, while we have been so careful about shielding all other commands from dumping their output to the screen as is.  If "online" logging is needed, provide a callback for it `log_stdout=callable`, which would log it into the log or do whatever you want it to do (e.g. UI could provide some kind of one line scroller or spinning wheel or whatever).

Reply:
Sounds like the better solution. Agree.

Reply:
If you agree, it likely means you understand what it means. Can you do it?

Reply:
Keep in mind that the output of annex init includes progress reports and counters. I guess that machinery can handle that too. Next application would then be to apply this to the non-existing output of GitPython's repoi init method. That progress reporting is also causing much confusing in the same context (slow network, slow FS).